crates/ra_ide/src/completion/complete_record_literal.rs

```diff
@@ -11,8 +11,24 @@ pub(super) fn complete_record_literal(acc: &mut Completions, ctx: &CompletionCon
         _ => return,
     };
 
+    let already_present_names: Vec<String> = ctx
+        .record_lit_syntax
+        .as_ref()
+        .and_then(|record_literal| record_literal.record_field_list())
+        .map(|field_list| field_list.fields())
+        .map(|fields| {
+            fields
+                .into_iter()
```

Comment:
```suggestion
        .map(|field_list|
            fields_list
                .fields()
                .into_iter()
```

Reply:
I don't see much of a difference tbh, so let's wait for somebody else with an opinion.